src/dpsvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/dfcsm/ValidationController.java

```diff
@@ -68,7 +70,16 @@ public GetValidOpenDFCMCase getValidOpenDFCMCase(@ApiParam(value = "driversLicen
             return GetValidOpenDFCMCase.ErrorResponse();
         }
 
-        return GetValidOpenDFCMCase.SuccessResponse("ROUTINE - PROFESSIONAL");
+        try {
+            CaseSequenceNumberResponse caseSequenceNumberResponse = ordsDfcmsApi.caseSequenceNumberGet(driversLicense, surcode);
```

Comment:
Did this work?? When I was working with the the FIGARO ORDS the response object was a JAVA 'Object' which needed to be cast into a LinkedHashMap and then be used to instantiate the response (in your case the CaseSequenceNumberResponse).

Reply:
yes @shaunmillargov, I changed the swagger to include an schema based on the response. We have to work with Carol on the fig one

Reply:
Good work.